test/integration/scheduler_perf/util.go

```diff
@@ -55,28 +58,35 @@ var dataItemsDir = flag.String("data-items-dir", "", "destination directory for
 // mustSetupScheduler starts the following components:
 // - k8s api server (a.k.a. master)
 // - scheduler
-// It returns clientset and destroyFunc which should be used to
+// It returns regular and dynamic clients, and destroyFunc which should be used to
 // remove resources after finished.
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
-func mustSetupScheduler() (util.ShutdownFunc, coreinformers.PodInformer, clientset.Interface) {
+func mustSetupScheduler() (util.ShutdownFunc, coreinformers.PodInformer, clientset.Interface, dynamic.Interface, *restmapper.DeferredDiscoveryRESTMapper) {
 	apiURL, apiShutdown := util.StartApiserver()
-	clientSet := clientset.NewForConfigOrDie(&restclient.Config{
+
+	cfg := &restclient.Config{
 		Host:          apiURL,
 		ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}},
 		QPS:           5000.0,
 		Burst:         5000,
-	})
-	_, podInformer, schedulerShutdown := util.StartScheduler(clientSet)
-	fakePVControllerShutdown := util.StartFakePVController(clientSet)
+	}
+
+	client := clientset.NewForConfigOrDie(cfg)
+	dynClient := dynamic.NewForConfigOrDie(cfg)
+	cachedClient := cacheddiscovery.NewMemCacheClient(client.Discovery())
+	restMapper := restmapper.NewDeferredDiscoveryRESTMapper(cachedClient)
```

Comment:
super nit: calculate `restMapper` outside of this function, as only one test cares about it.

Reply:
done.